src/api/version.ts

```diff
@@ -132,6 +133,20 @@ export namespace VersionHelper {
     return stdout.split('\n').filter(value => value.startsWith(versionPrefix)).map(value => value.substring(versionPrefix.length))[0]
   }
 
+  export async function getCheVersionWithKubectl(namespace: string, podName: string, versionPrefix: string): Promise<string | undefined> {
```

Comment:
It is not needed to separate into two functions depending on client
 https://github.com/che-incubator/chectl/blob/master/src/util.ts#L18